### ISSUE TITLE
Force msg toggle for a player

### ISSUE THREAD

**Issue by AymerickA** [AuraDevelopmentTeam/BungeeChat2#158]
It's not a bug just a suggestion.

Add a command to force the message private toggle of a player (compatible with console also)

So add 1 argument for command /msgtoggle <player> with a new permission admin needed if a argument is write.

So for just /msgtoggle the permission the existing existing permission is sufficient
But for /msgtoggle <player> need a permission admin.

Thanks to considerer this suggestion.

**Comment by AwesomestCode** [comment]
If I understand you correctly, you wish for a player to be able to toggle their own status of accepting or not accepting private messages with the same permission as we currently do, but have a separate permission if you are toggling for another user.

Also, do you wish for there to be a direct "bypass" for administrators if the intended recipient is not accepting private messages?

I'm asking this because if I understand correctly, I may implement/PR this suggestion.

**Comment by AymerickA** [comment]
"If I understand you correctly, you wish for a player to be able to toggle their own status of accepting or not accepting private messages with the same permission as we currently do, but have a separate permission if you are toggling for another user." Exactly.

"Also, do you wish for there to be a direct "bypass" for administrators if the intended recipient is not accepting private messages?" You want say for administrator can change message toggle for other administrators with a permission to can't toggle message ?

And if you could develop that it would be great it would help the developer in the big task list he seems to have, thank you.

**Comment by AwesomestCode** [comment]
No, I meant if there would be a way for the admin to not have to change the msgtoggle settings and there would be a permission so that msgtoggle settings of the recipient are ignored.

**Comment by AymerickA** [comment]
Hmm... I don't understand.

I understood as you say and I think, a bypass permission like "bungeechat.bypass.msgtoggle" so that a player can't get his msgtoggle changed.

**Comment by BrainStone** [comment]
@AwesomestCode there is already a bypass permission: ``bungeechat.chat.bypass.toggle``

**Comment by AymerickA** [comment]
@BrainStone Nice, but need a command to toggle the msg of other player

**Comment by BrainStone** [comment]
I understand that. It's just the other thing Awesome was talking about is already implemented.

**Comment by AymerickA** [comment]
Ok perfect thanks.

I want to say that you've always responded to my issues, you're a good and active developer.

**Comment by BrainStone** [comment]
I do my very best and thank you for recognizing my efforts.

**Comment by AymerickA** [comment]
Sorry to come back like this, but I think this feature would be useful and take I THINK little development time please

**Comment by BrainStone** [comment]
It’s already implemented.

**Comment by AymerickA** [comment]
How please ?

**Comment by AwesomestCode** [comment]
Look up. I made a PR with code that implements it.

**Comment by BrainStone** [comment]
Try this file: https://maven.project-creative.net/repository/auradev-snapshots/dev/aura/bungeechat/BungeeChat/2.2.4.1263-DEV/BungeeChat-2.2.4.1263-DEV.jar
